src/vfc_precexp

```diff
@@ -350,24 +356,49 @@ def search_minimum_arguments(	runPath,
 
 
 if __name__ == "__main__":
+    
+    parser = argparse.ArgumentParser(description="The vfc_precexp script tries "
+                                     "to minimize the precision for each function "
+                                     "call of a code compiled with verificarlo")
+    parser.add_argument("runPath", help="command used to execute the target program")
+    parser.add_argument("cmpPath", help="command used to compare an execution with the reference")
+    
+    parser.add_argument("--maxTimeout", type=int, default=None,
+                        help="maximum execution time (ms)")
+    parser.add_argument("-m", "--mode", choices=["arguments", "operations", "all"], default="all",
+                        help="function instrumentation mode")
+    parser.add_argument("--vprec_mode", choices=["ieee", "ib", "ob", "all"], default="ob",
+                        help="vprec operating mode")
+    
+    args = parser.parse_known_args()
 
     if len(sys.argv) < 3:
-        print("explore.py [exrun] [excmp] functions ...")
+        print("explore.py [runPath] [cmpPath] functions ...")
         exit()
 
-    runPath = sys.argv[1]
-    cmpPath = sys.argv[2]
-    function_list = []
-
-    for i in range(3, len(sys.argv)):
-        function_list.append(sys.argv[i])
+    # runPath = sys.argv[1]
```

Comment:
Please remove these commented lines.

Reply:
I realized this issue just before you replied and had already committed the cleaned-up version.

Reply:
Excellent. I let you merge then.